.github/workflows/linter-conan-v2.yml

```diff
@@ -1,124 +0,0 @@
-        uses: mikefarah/yq@master
-        with:
-          cmd: yq '.conan.version' '.c3i/config_v1.yml'
-
-      - uses: actions/setup-python@v4
-        if: steps.changed_files.outputs.any_changed == 'true'
-        with:
-          python-version: ${{ env.PYVER }}
-
-      - name: Install requirements
-        if: steps.changed_files.outputs.any_changed == 'true'
-        run: |
-          pip install ${{ env.REQUIREMENTS }} conan==${{ steps.parse_conan_v1_version.outputs.result }}
-
-      - name: Execute linter over all recipes in the repository
-        id: linter_recipes
-        if: steps.changed_files.outputs.any_changed == 'true'
-        run: |
-          echo '## Linter summary (recipes)' >> $GITHUB_STEP_SUMMARY
-          pylint --rcfile=linter/pylintrc_recipe `ls recipes/*/*/conanfile.py | shuf -n 500` --output-format=json --output=recipes.json --score=y --exit-zero
-          jq '[map( select(.type=="error")) | group_by (.message)[] | {message: .[0].message, length: length}] | sort_by(.length) | reverse' recipes.json > recipes2.json
-          jq -r '.[] | " * \(.message): \(.length)"' recipes2.json >> $GITHUB_STEP_SUMMARY
-
-      - name: Execute linter over all test_package/recipes in the repository
-        id: linter_test_package
-        if: steps.changed_files.outputs.any_changed == 'true'
-        run: |
-          echo '## Linter summary (test_package)' >> $GITHUB_STEP_SUMMARY
-          pylint --rcfile=linter/pylintrc_testpackage `ls recipes/*/*/test_package/conanfile.py | shuf -n 500` --output-format=json --output=recipes.json --exit-zero
-          jq '[map( select(.type=="error")) | group_by (.message)[] | {message: .[0].message, length: length}] | sort_by(.length) | reverse' recipes.json > recipes2.json
-          jq -r '.[] | " * \(.message): \(.length)"' recipes2.json >> $GITHUB_STEP_SUMMARY
-
-  conanfile_recipe:
-    name: Lint changed conanfile.py (v2 migration)
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v3
-      - name: Get changed files
-        id: changed-files
-        uses: ./.github/actions/pr_changed_files
-        with:
-          files: |
-            recipes/*/*/conanfile.py
-      - name: Get Conan v1 version
-        id: parse_conan_v1_version
-        if: steps.changed-files.outputs.any_changed == 'true'
-        uses: mikefarah/yq@master
-        with:
-          cmd: yq '.conan.version' '.c3i/config_v1.yml'
-      - uses: actions/setup-python@v4
-        if: steps.changed-files.outputs.any_changed == 'true'
-        with:
-          python-version: ${{ env.PYVER }}
-      - name: Install dependencies
-        if: steps.changed-files.outputs.any_changed == 'true'
-        run: |
-          pip install ${{ env.REQUIREMENTS }} conan==${{ steps.parse_conan_v1_version.outputs.result }}
-      - name: Run linter
-        if: steps.changed-files.outputs.any_changed == 'true'
-        run: |
-          echo "::add-matcher::linter/recipe_linter.json"
```

Comment:
The file `linter/recipe_linter.json` is only used by this file. So it could be removed as well.